docs/src/functions_overview.md

```diff
@@ -0,0 +1,69 @@
+# Functions
+Here the *Special Functions* are listed according to the structure of [NIST Digital Library of Mathematical Functions](https://dlmf.nist.gov/).
+
+## Error Functions, Dawson’s and Fresnel Integrals
+
+| Function | Description |
+|:-------- |:----------- |
+| [`erf(x)`](@ref SpecialFunctions.erf)         | [error function](https://en.wikipedia.org/wiki/Error_function) at ``x`` |
+| [`erfc(x)`](@ref SpecialFunctions.erfc)       | complementary error function, i.e. the accurate version of ``1-\operatorname{erf}(x)`` for large ``x`` |
+| [`erfcinv(x)`](@ref SpecialFunctions.erfcinv) | inverse function to [`erfc()`](@ref SpecialFunctions.erfc) |
+| [`erfcx(x)`](@ref SpecialFunctions.erfcx)     | scaled complementary error function, i.e. accurate ``e^{x^2} \operatorname{erfc}(x)`` for large ``x`` |
+| [`erfi(x)`](@ref SpecialFunctions.erfi)       | imaginary error function defined as ``-i \operatorname{erf}(ix)`` |
+| [`erfinv(x)`](@ref SpecialFunctions.erfinv)   | inverse function to [`erf()`](@ref SpecialFunctions.erf) |
+| [`dawson(x)`](@ref SpecialFunctions.dawson)   | scaled imaginary error function, a.k.a. Dawson function, i.e. accurate ``\frac{\sqrt{\pi}}{2} e^{-x^2} \operatorname{erfi}(x)`` for large ``x`` |
+
+
+## Elliptic Integrals
+
+| Function | Description |
+|:-------- |:----------- |
+| [`ellipk(m)`](@ref SpecialFunctions.ellipk)    | [complete elliptic integral of 1st kind](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants) ``K(m)``  |
+| [`ellipe(m)`](@ref SpecialFunctions.ellipe)    | [complete elliptic integral of 2nd kind](https://en.wikipedia.org/wiki/Elliptic_integral#Complete_elliptic_integral_of_the_second_kind) ``E(m)`` |
+
+
+# WIP not grouped
+| Function                                                      | Description                                                                                                                                                     |
```

Comment:
I would suggest the following groupings:
- Airy, Bessel and Hankel functions
- Gamma and related functions (anything involving `gamma`, `beta`, `logfactorial`)
- Trigonmetric integral (`sinint`, `cosint`)
- Riemann zeta and related functions (`zeta`, `eta`)

Reply:
Sounds good. I'm doing the trigonometric integrals right now. Will also group the other ones but their documentation will take more time..